tests/conftest.py

```diff
@@ -285,6 +296,17 @@ def assert_collapsed(runtime: str) -> Callable[[Mapping[str, int], bool], None]:
     return partial(assert_function_in_collapsed, function_name)
 
 
+@fixture
+def assert_application_name(application_pid: int, runtime: str, in_container: bool) -> Generator:
+    desired_names = {
+        "java": "java: /app/Fibonacci.jar",
+        "python": "python: /app/lister.py",
+    }
+    yield
```

Comment:
Why `yield` before the check?

Reply:
Hmm I would like to check it only after the tests are done so we don't miss maybe some required warm-up/wait, etc..

Reply:
Explain in comment